app/Resources/views/program.html.twig

```diff
@@ -95,7 +95,7 @@
         <span>
           {{ "description"|trans({}, "catroweb") }}
           {% if app.user and my_program %}
-            <a href="#" data-toggle="tooltip" title="Change the program Description">
+            <a href="#" data-toggle="tooltip" title="{{ 'programs.editDescription'|trans({}, 'catroweb') }}">
               <i id="edit-description-button" class="ml-3 fa fa-pencil-square-o"></i>
```

Comment:
Maybe remove the effect of making the button larger on hover. Not sure if we should have that but might be a bit inconsistend compared to the button to edit ones profile...